sonic_data_client/mixed_db_client.go

```diff
@@ -160,6 +159,17 @@ func getZmqClientByAddress(zmqAddress string) (swsscommon.ZmqClient, error) {
 	return client, nil
 }
 
+func removeZmqClient(zmqClient swsscommon.ZmqClient) (error) {
+	for address, client := range zmqClientMap {
+		if client == zmqClient { 
```

Comment:
>zmqClient [](http://example.com/codeflow?start=15&length=9)

we need to handle resource dispose aggressively. not a blocking issue right now.